src/MfaVerifyController.js

```diff
@@ -133,10 +133,7 @@ export default BaseLoginController.extend({
       }
     });
 
-    if (!this.settings.get('features.hideSignOutLinkInMFA') &&
-        !this.settings.get('features.mfaOnlyFlow')) {
-      this.add(new FooterSignout(this.toJSON()));
-    }
+    this.add(new FooterMFA(this.toJSON()));
```

Comment:
do you have to pass `this.toJSON()` ?

Reply:
1. I didn't look into okta core, but looks like we need to pass `this.toJSON()` to okta views constructor
2. Other controllers are extended from FormController and can use `Footer` property: https://github.com/okta/okta-signin-widget/blob/master/src/util/FormController.js#L98
MfaVerifyController is extended from BaseLoginController, which don't have such functionality

Reply:
In `VerifyCustomFactorController`, you did `this.addFooter(FooterMFA);`, looks like it's working?